lib/ts/utils.ts

```diff
@@ -207,28 +208,62 @@ export function validateTheStructureOfUserInput(
     }
 }
 
-export async function getAPIVersionFromFileIfExists(): Promise<string | null> {
+export async function getDataFromFileIfExists<T>(filePath: string): Promise<T | undefined> {
     try {
-        let apiVersionFromFile = await new Promise<Buffer>((resolve, reject) => {
-            readFile(API_VERSION_FILE_PATH, (err, data) => {
+        let dataFromFile = await new Promise<Buffer>((resolve, reject) => {
+            readFile(filePath, (err, data) => {
                 if (err !== undefined && err !== null) {
                     reject(err);
                 }
                 resolve(data);
             });
         });
-        return apiVersionFromFile.toString();
+        return JSON.parse(dataFromFile.toString());
     } catch (err) {
-        return null;
+        return undefined;
     }
 }
 
-export async function storeAPIVersionInFile(apiVersion: string) {
+export async function storeIntoTempFile(filePath: string, data: any) {
```

Comment:
Change the name of the function to be `storeIntoTempFolderForServerlessCache`

Reply:
done